pkg/config/services.go

```diff
@@ -352,6 +372,7 @@ var SupportedServices = serviceConfigs{
 			regexp.MustCompile("cluster:(?P<CacheClusterId>[^/]+)"),
 			regexp.MustCompile("serverlesscache:(?P<clusterId>[^/]+)"),
 		},
+		AvailableEnhancedMetrics: elasticache.NewElastiCacheService(nil).ListSupportedEnhancedMetrics(),
```

Comment:
It feels like this should work without constructing the service.

Reply:
Agreed, please feel free to remove this change and in the future we'll make it part of the registry to create a map of all supported services with their supported enhanced metrics

Reply:
Removed